src/main/java/com/tenx/ms/retail/product/domain/ProductEntity.java

```diff
@@ -0,0 +1,38 @@
+package com.tenx.ms.retail.product.domain;
+
+import com.tenx.ms.retail.stock.domain.StockEntity;
+import com.tenx.ms.retail.store.domain.StoreEntity;
+import lombok.Data;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+@Data
+@Table(name = "product")
+public class ProductEntity {
+    @Id
+    @GeneratedValue
+    @Column(name="product_id", nullable = false)
+    private Long productId;
+
+    @Column(name="name", nullable = false)
+    private String name;
+
+    @Column(name = "description", nullable = false)
+    private String description;
+
+    @Column(name = "sku", nullable = false, length = 10)
+    private String sku;
+
+    @Column(name = "price", nullable = false)
```

Comment:
@alfredo-github I think that this is a convention that they adopted throughout their dev process. I've seen it in almost any old or new MS.